docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+---
+sidebar_label: "Configure Applications to Use Proxy Server"
+title: "Configure Applications to Use Proxy Server"
+description: "Guide to configure configurations in a cluster to use a proxy server."
+hide_table_of_contents: false
+sidebar_position: 201
+tags: ["clusters", "cluster management", "proxy"]
+---
+
+You can configure applications that operate in your Palette clusters to use a proxy server to access the internet
+instead of accessing the internet directly.
+
+Before you are able to configure applications in a cluster to use the proxy server, you must first ensure that the
+cluster itself is configured to use the proxy server. This means that the host Operating System (OS) and the Palette
+agent inside the cluster will use the proxy server for its outbound communications. The steps to do this vary depending
+on your environment.
+
+After the cluster is configured to use the proxy server, you can proceed to configure the applications inside the
+cluster to use the proxy server. You can do this by applying the `spectrocloud.com/connection: proxy` label to the
+specific job, deployment, or daemon set to instruct an application to use the proxy settings of the cluster.
+
+## Prerequisites
+
+- An active proxy server reachable by your cluster.
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+- A PCG is deployed into an active and healthy Kubernetes cluster. Refer to
```

Comment:
You can either to deploy to an existing k8s cluster, or use the Palette CLI. However, keep in mind that the CLI can only be used to deploy PCGs to OpeStack, Vsphere, or MAAS. All other platforms require the the existing k8s cluster install method through Helm.  So we should link to either the k8s deploy guide or the index page to the CLI install [page](https://docs.spectrocloud.com/clusters/pcg/deploy-pcg/).